app/app.go

```diff
@@ -556,6 +572,7 @@ func New(
 		// TODO: Check alternatives to this
 		// AddRoute(distrtypes.RouterKey, distr.NewCommunityPoolSpendProposalHandler(app.DistrKeeper)).
 		AddRoute(upgradetypes.RouterKey, upgrade.NewSoftwareUpgradeProposalHandler(&app.UpgradeKeeper)).
+		// AddRoute(ibcexported.RouterKey, ibcclient.NewClientProposalHandler(app.IBCKeeper.ClientKeeper))
```

Comment:
should it be in a comment? these lines didn't exist before

Reply:
yes, currently we dont enable this, maybe in the future